code/datums/mutantraces.dm

```diff
@@ -100,6 +98,19 @@ TYPEINFO(/datum/mutantrace)
 	/// If our mutant has a female variant that has different organs, these will be used instead
 	var/list/mutant_organs_f = null
 
+	/// icon definitions for mutantrace clothing variants. one icon file per slot.
```

Comment:
```suggestion
	// icon definitions for mutantrace clothing variants. one icon file per slot.
```

Reply:
Removed in [0ca96b9](https://github.com/goonstation/goonstation/pull/9773/commits/0ca96b9e2da60a73de054e02e5e442837ff98a2e)